docs/content/latest/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ysql.md

```diff
@@ -39,31 +39,21 @@ Use Yugabyte Platform to back up your YugabyteDB universe YSQL data.
 
 To schedule backups for a later time, or as a recurring task, see [Schedule universe YSQL data backups](../../schedule-data-backups/ysql).
 
-To immediately back up your YugabyteDB universe YSQL data:
+To immediately back up your YugabyteDB universe YSQL data, perform the following:
 
-1. Open the **Universe Overview** and then click the **Backups** tab. The **Backups** page appears.
+1. Open your universe and select **Backups**.
 1. Click **Create Backup** to open the **Create Backup** dialog.
 
     <br/><br/>
 
-    ![Create Backup - YSQL](/images/yp/create-backup-ysql.png)
+    ![Create Backup - YSQL](/images/yp/create-backup-ysql.png)<br><br>
 
-1. Click the **YSQL** tab and enter the following information:
+1. Complete the fields presented in the **YSQL** tab.
 
-    - **Storage**: Select the storage type: `GCS Storage`, `S3 Storage`, or `NFS Storage`.
-    - **Namespace**: Select the namespace from the drop-down list of available namespaces.
-    - **Parallel Threads**: Enter or select the number of threads. The default is `8`.
+1. Click **OK** to start the requested backup immediately.
 
-1. Click **OK**. The requested backup begins immediately.
+If the universe has [encryption at rest enabled](../../../security/enable-encryption-at-rest), data files are backed up as-is (encrypted) to reduce the computation cost of a backup and to keep the files encrypted. A universe key metadata file, containing key references, is also backed up. To allow Yugabyte Platform to back up your data with the user authentication enabled, follow the instructions provided in [Edit configuration flags ](../../../manage-deployments/edit-config-flags) to add the `ysql_enable_auth=true` and `ysql_hba_conf_csv="local all all trust"` YB-TServer flags.
 
-{{< note title="Note" >}}
+Note that currently Yugabyte Platform has one limitation to its backup functionality due to the lack of support for enums. To mitigate the issue, it is recommended that you use the [ysql_dump](https://docs.yugabyte.com/latest/admin/ysql-dump/) utility and then `/COPY` action for your backups.
```

Comment:
```suggestion
Note that currently Yugabyte Platform has one limitation to its backup functionality. If your database, managed by the Platform, has enums, please use the [ysql_dump](https://docs.yugabyte.com/latest/admin/ysql-dump/) utility to take backups and `/COPY` action to restore.
```

Reply:
@streddy-yb , please take a look.

Reply:
hi @lizayugabyte - Aayush's fix for PLAT-2591 has already landed in 2.11.2, 2.8, and 2.6 branches. I suggest we rephrase the above statement as follows:
Note that older versions of Yugabyte Platform do not support taking backups of YSQL databases that make use of **enum** types. This limitation is fixed in 2.6.11.0, 2.8.2.0, 2.11.2.0 and newer releases. If you're currently using an older version of Platform that has this limitation, you can use ysql_dump (https://docs.yugabyte.com/latest/admin/ysql-dump/) utility directly along with  `/COPY` action to workaround this issue.

Reply:
Hi @streddy-yb Since the customers would see a specific version of the doc by selecting the release on the doc site, I don't think we need this note in 2.6, 2.8, 2.11 because it's a non-issue for the customers. I'd add the note to 2.4 and 2.2 docs. Please let me know if you are okay with this solution.

Reply:
Added a note to latest, stable, and 2.6 doc streams.